src/submission/validation-clinical/followUp.ts

```diff
@@ -24,13 +24,82 @@ import {
 	SubmissionValidationOutput,
 	DonorVitalStatusValues,
 } from '../submission-entities';
-import { ClinicalEntitySchemaNames, FollowupFieldsEnum } from '../../common-model/entities';
+import {
```

Comment:
changes here mirror changes to other validation files.
big diff hash because I moved an inline function out of a function body